frontend/app/assets/javascripts/archival_objects.crud.js

```diff
@@ -0,0 +1,302 @@
+//= require agents.crud
+//= require subjects.crud
+//= require dates.crud
+//= require notes.crud
+//= require instances.crud
+//= require lang_materials.crud
+//= require deaccessions.crud
+//= require subrecord.crud
+//= require rights_statements.crud
+//= require form
+
+function SimpleLinkingModal(config) {
+  var self = this;
+  self.config = config;
+  self.page = 1;
+  self.currentSelected = undefined;
+  self.context_filter_term = [];
+  (config.context_filter_term || []).forEach((element, index) => {
+    self.context_filter_term.push(JSON.stringify(element));
+  });
+  self.config.label_plural = config.button_title;
+  self.$modal = AS.openCustomModal(
+    'linkResourceModal',
+    self.config.title,
+    AS.renderTemplate('linker_browsemodal_template', config),
+    'large',
+    {},
+    this
+  );
+  if (config.must_select) {
+    $('.modal-header a', self.$modal).hide();
```

Comment:
This line hides the first modal's `x` close button, yet the user can still cancel the modal via the escape key or clicking outside of the modal. This line also makes the first modal inconsistent with all other modals since they all have the `x` close button. Might be better to remove this line to give users an explicit close button in case they click the "spawn archival object" link by mistake, etc.

Reply:
Added the `x` close button back to the resource modal. thx.